packages/pwa-kit-react-sdk/src/ssr/server/react-rendering.js

```diff
@@ -284,6 +293,7 @@ const renderApp = (args) => {
             extractor
         )
     }
+    performance.mark(PERFORMANCE_MARKS.renderToStringEnd)
```

Comment:
How about placing this marker to later in the `renderApp` function? There's another `renderToString` called:  https://github.com/SalesforceCommerceCloud/pwa-kit/blob/1dcf49a38f57d665b8f82f5ed297c61dbba0906a/packages/pwa-kit-react-sdk/src/ssr/server/react-rendering.js#L378-L387

Reply:
fixed in new PR